Code/Tools/FBuild/FBuildTest/Data/TestCSharp/csharp.bff

```diff
@@ -8,9 +8,10 @@
 Using( .StandardEnvironment )
 Settings {}
 
-Alias( 'CSharpCompiler' )
+Compiler( 'CSharpCompiler' )
```

Comment:
Changed the CS test to actually use a compiler instead of a FileNode with an alias. This may break people's projects. Any ideas?